components/Sanitize/Sanitize.jsx

```diff
@@ -0,0 +1,8 @@
+import sanitizeHtml from 'sanitize-html'
```

Comment:
Please use a semantical name like: <TextContent>

Reply:
changed to sanitize